README.md

```diff
@@ -32,21 +32,21 @@
 
 # Overview
 
-IoTDB (Internet of Things Database) is an integrated data management engine designed for time series data, which can provide users specific services for data collection, storage and analysis. Due to its light weight structure, high performance and usable features together with its intense integration with Hadoop and Spark ecology, IoTDB meets the requirements of massive dataset storage, high-speed data input and complex data analysis in the IoT industrial field.
+IoTDB (Internet of Things Database) is an integrated data management engine designed for time series data, which can provide users specific services for data collection, storage and analysis. Due to its light weight structure, high performance and usable features together with its intense integration with Hadoop and Spark ecology, IoTDB meets the requirements of massive dataset storage, high throughput data input and complex data analysis in the industrial IoTDB field.
 
 # Main Features
 
-IoTDB's features are as following:
+Main features of IoTDB are as follows:
 
-1. Flexible deployment. IoTDB provides users one-click installation tool on the cloud, once-decompressed-used terminal tool and the bridge tool between cloud platform and terminal tool (Data Synchronization Tool).
-2. Low cost on hardware. IoTDB can reach a high compression ratio of disk storage
+1. Flexible deployment strategy. IoTDB provides users one-click installation tool on the cloud, one-shot compression terminal tool and a data synchronization tool bridging cloud platform and terminal.
```

Comment:
Hm... I can not understand the original and the modified sentences....
I think if we want to highlight the "flexible deployment strategy" feature, we need to show two points:
1. IoTDB provides users one-click installation tool  either on the cloud or the terminal device (maybe "embedded device" is better.) (users just need to using docker command or  unzip IoTDB's zip file to enjoy IoTDB).
2. We provide interactive CLI tool to operate IoTDB, and a data synchronization tool bridging cloud platform and terminal device (or embedded device).